clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -259,8 +262,49 @@ public T records() {
         }
     }
 
+    public static final class IdError {
```

Comment:
Since we have session ids and topic ids in the context of a fetch request, we should probably qualify `TopicId`